veggieseasons/lib/screens/details.dart

```diff
@@ -4,18 +4,153 @@
 
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/widgets.dart';
+import 'package:scoped_model/scoped_model.dart';
+import 'package:veggieseasons/data/model.dart';
+import 'package:veggieseasons/data/veggie.dart';
 import 'package:veggieseasons/styles.dart';
 
+class SeasonCircle extends StatelessWidget {
```

Comment:
Add a class doc for what this is supposed to be

Reply:
Done.